website/public/locales/de/common.json

```diff
@@ -22,6 +23,8 @@
   "faq": "FAQ",
   "guidelines": "Richtlinien",
   "github": "GitHub",
+  "hide": "Hide",
+  "hugging_face": "HuggingFace",
```

Comment:
I dont know why we translate this lol

Reply:
Just part of the general structure I guess. But I forgot to translate hide and copy here, ups.